crates/red_knot_python_semantic/resources/mdtest/protocols.md

```diff
@@ -13,3 +13,13 @@ def _(some_int: int, some_literal_int: Literal[1], some_indexable: SupportsIndex
     b: SupportsIndex = some_literal_int
     c: SupportsIndex = some_indexable
 ```
+
+## Invalid
+
+```py
+from typing import Protocol
+
+# error: [invalid-type-form] "`typing.Protocol` requires at least one argument when used in a type expression"
```

Comment:
This error message should just say "`typing.Protocol`" cannot be used in a type expression"